Python/ceval.c

```diff
@@ -4638,6 +4628,7 @@ _PyEval_EvalFrameDefault(PyThreadState *tstate, InterpreterFrame *frame, int thr
                         stack_pointer, nargs, kwnames
                     );
                     STACK_SHRINK(postcall_shrink);
+                    postcall_shrink = 1;
```

Comment:
Note to self: this (and other similar lines below) is done to reset `postcall_shrink` to the default value of 1 for the next eval frame/loop.